### ISSUE TITLE
Brave's ad-block's region_list is not as complete as uBO's

### ISSUE THREAD

**Issue by rht** [brave/ad-block#146]
Missing regions are NOR, ROU, POR, and THA.

**Comment by krystian3w** [comment]
Also Polish moved anti-adblock filters to other repository:

https://github.com/olegwukr/polish-privacy-filters/blob/master/adblock.txt
https://github.com/olegwukr/polish-privacy-filters/blob/master/anti-adblock-suplement.txt

**Comment by pes10k** [comment]
I defer to @ryanbr if we should pull in new lists.  If he says so, this is 👍 from me.  

@ryanbr what do you think?

**Comment by ryanbr** [comment]
Okay, will look through these lists. Some are very specific to uBO (which won't work in Brave Release)

**Comment by ryanbr** [comment]
I don't see an issue with it, though applying possibly all separate Anti-adblock lists for all languages would make `brave://adblock` much larger (which may need to reorganise if got too large). But as its stands, it's fine to include. If @pes10k is okay, I can patch this in this week.

**Comment by pes10k** [comment]
@ryanbr sounds good!  If you need any help handling, let me know.  Also, @bbondy , is it alright ot add these additional languages in?

**Comment by bbondy** [comment]
you can go ahead and add more languages 👍

**Comment by pes10k** [comment]
terrific!  @ryanbr if you need any help with this let me know. Otherwise, looks like you're good to go!

**Comment by krystian3w** [comment]
Also see this: https://github.com/brave/adblock-lists/issues/334

Prebake is unsupported (the repository has been archived and has not been updated for several years), no block many cookies bars / popups and possible generate breakge.

https://github.com/brave/ad-block/blob/0a03587e37990ea5c22abca798d8c63da06bf3e3/lists/regions.h#L88

**Comment by ryanbr** [comment]
@krystian3w Yeah, we're aware of it. https://github.com/brave/adblock-rust/pull/67  (WIP)
